vmm/src/config.rs

```diff
@@ -278,6 +281,8 @@ impl MemoryConfig {
                 mergeable_str = &param[10..];
             } else if param.starts_with("hotplug_size=") {
                 hotplug_str = &param[13..]
+            } else if param.starts_with("balloon=") {
```

Comment:
Here I have a genuine question, I'm not sure how balloon is supposed to be handled. Is this another way of doing memory hotplug? It's important to understand what's the expectation is to understand if it should be added the same way `virtio-mem` is being added.
/cc @rbradford @sameo

Reply:
It's not doing memory hotplug, but I think it makes sense to add it as part of the memory configuration (and not as a standalone `virtio-ballon` switch).